src/oc/digest/resources/user.clj

```diff
@@ -40,7 +40,7 @@
 
   ([user :guard #(= (keyword (:digest-medium %)) :slack)] true) ; Slack user
 
-  ([user :guard #(= (keyword (:status %)) :active)] true) ; active email user
+  ([user :guard #(and (= (keyword (:digest-medium %)) :email) = (keyword (:status %)) :active)] true) ; active email user
```

Comment:
@bago2k4 does this syntax work? you have a bare `=` after that first and condition